models/_sector/nft/trades/solana/nft_solana_trades_schema.yml

```diff
@@ -0,0 +1,163 @@
+version: 2
+
+models:
+  - name: nft_solana_trades
+    meta:
+      blockchain: solana
+      sector: nft
+      contributors: [ilemi]
+    config:
+      tags: ['magiceden','tensorswap','solana','nft']
+    description: >
+       nft solana trades
+    # tests:
+    #  - dbt_utils.unique_combination_of_columns:
+    #       combination_of_columns:
+    #         - project
+    #         - trade_category
+    #         - outer_instruction_index
+    #         - inner_instruction_index
+    #         - account_metadata
+    #         - tx_id
```

Comment:
you can go ahead and remove this. as long as it remains materialized as a view, no need here, only needed in project-level spells/schemas